docs/source/parsing.md

```diff
@@ -25,30 +25,39 @@ To parse a collection into a forward index, use `parse_collection` command:
 
 For example:
 
+    $ mkdir -p path/to/forward
     $ zcat ClueWeb09B/*/*.warc.gz | \   # pass unzipped stream in WARC format
         parse_collection \
         -j 8 \                          # use up to 8 threads at a time
         -b 10000 \                      # one thread builds up to 10k documents in memory
         -f warc \                       # use WARC
         --stemmer porter2 \             # stem every term using the Porter2 algorithm
         --content-parser html \         # parse HTML content before extracting tokens
-        -o cw09b
+        -o path/to/forward/cw09b
 
-The above command will write the following files:
-- `cw09b`: forward index in binary format,
+In case you get the error `-bash: /bin/zcat: Argument list too long`, you can pass the unzipped stream using:
+
+    $ find ClueWeb09B -name '*.warc.gz' -exec zcat -q {} \;
+
+The parsing will write the following files:
+- `cw09b`: forward index in binary format.
 - `cw09b.terms`: a new-line-delimited list of sorted terms,
-  where term having ID N is on line N
+  where term having ID N is on line N+1.
```

Comment:
I think it might be more clear if we stick to `N` and add something like: "(indexed from 0)" because now it suggests there's some kind of shift between ID and a line. @amallia what do you think?

Reply:
Maybe it could be:
```suggestion
  where term on line N has the ID N-1.
```

Reply:
Maybe it could be: "where the term on line N has the ID N-1".

Reply:
Maybe it could be: "where term on line N has the ID N-1".

Reply:
I see, what do you think about "term having ID N is on line N, with N starting from 0"?

Reply:
This sounds fine.